roles/handle_c2r_report/tasks/check-results-file.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Result file status
+  ansible.builtin.stat:
+    path: "{{ handle_c2r_report_result_filename_txt }}"
+  register: result
+- name: Print the value of a variable with message
+  ansible.builtin.debug:
+    msg: "The value of my_variable is: {{ handle_c2r_report_result_filename_txt }}"
+- name: Set variable
+  ansible.builtin.set_fact:
+    result_file_exists: result.stat.exists
+- name: Check that result file exists
+  ansible.builtin.assert:
+    that: result.stat.exists
+    msg: "The analysis report file {{ handle_c2r_report_result_filename_txt }} was not created."
```

Comment:
This would need to be either analysis or conversion report, no?

Reply:
Yeah, the path of `handle_c2r_report_result_filename_txt` is being changed based on activity

Reply:
Yeah, but the `The analysis report file ` part...

Reply:
I see, I removed the `analysis` from the message.